fiboa_cli/datasets/fs_br.py

```diff
@@ -0,0 +1,206 @@
+from ..convert_utils import convert as convert_
+import re
+import pandas as pd
+
+# Please REMOVE the comment after review: 
+# I was actually using this link "https://prod-dcd-datasets-cache-zipfiles.s3.eu-west-1.amazonaws.com/vz6d7tw87f-1.zip" which is copied from the 
+# "Download All" button from the website (https://data.mendeley.com/datasets/vz6d7tw87f/1). 
+# But output an error saying """/vsizip//tmp/tmp7puy6g90/vz6d7tw87f-1.zip' not recognized as a supported file format. 
+# It might help to specify the correct driver explicitly by prefixing the file path with '<DRIVER>:', e.g. 'CSV:path'""""
+SOURCES = "https://github.com/fiboa/data/files/15438544/LEM_dataset.zip"
+
+
+ID = "br"
+SHORT_NAME = "Brazil"
+TITLE = "Field boundaries for Brazil"
```

Comment:
Maybe we should indicate that this is not all of Brazil, but only (parts of) the Bahia state, so

```suggestion
ID = "br_ba"
SHORT_NAME = "Brazil (Bahia)"
TITLE = "Field boundaries for Brazil (parts of Bahia state)"
```

This would also require a rename of the file to "br_ba.py".

Reply:
But it's very far from all of Bahia. I think I'd do the shortname in line with the paper - https://data.mendeley.com/datasets/vz6d7tw87f/1#file-5ac1542b-12ef-4dce-8258-113b5c5d87c9

Shortname of 'LEM+ dataset', Title: "Monthly field boundaries and land use information about 1854 fields from Luís Eduardo Magalhães (LEM) and other municipalities in the west of Bahia state, Brazil"

Reply:
yeah, maybe "lem_br_ba" etc?

Reply:
I will change it to "br_lem_ba" then

Reply:
While I was at it, I updated the file with a proposal. Feel free to change if you think it's not good.

Reply:
I use lem as prefix because we also did put the organization first for other converters, e.g. ec_

Reply:
Oh wait, my fault. It should actually be br_ba_lem, because LEM is the acronym for the municipalities...